clients/mobile/app.json

```diff
@@ -12,9 +12,35 @@
       "backgroundColor": "#ffffff"
     },
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.example.mobile",
+      "infoPlist": {
+        "CFBundleURLTypes": [
+          {
+            "CFBundleURLSchemes": [
+              "myapp"
+            ]
+          }
+        ]
+      }
     },
     "android": {
+      "package": "com.example.mobile",
```

Comment:
same as aboce

Reply:
Updated the `package` to `net.bloodconnect.app` to match our domain `bloodconnect.net` instead of `com.example.mobile`.